### PR TITLE
Require ratings when adding new submission, fixed spec test

### PR DESCRIPTION
**Pull request by Jose-N**

### CODE REVIEW

File: app/controllers/submissions_controller.rb [hunk 1]
```diff
@@ -51,11 +60,19 @@ def set_submission
     end
```
Comment (kyjoya) [inline app/controllers/submissions_controller.rb]: you'll want to make these private methods

File: app/views/submissions/show.html.erb [hunk 2]
```diff
@@ -90,7 +90,7 @@
 
 <h4>Comments</h4>
 <ul>
-  <% @submission.ratings.each do |rating|%>
+  <% @submission.ratings.drop(1).each do |rating|%>
```
Comment (kyjoya) [inline app/views/submissions/show.html.erb:93]: why is it necessary to drop the first rating?

File: app/views/submissions/show.html.erb [hunk 3]
```diff
@@ -23,7 +23,7 @@
 <%= " posted @ #{@submission.created_at} by" %> <strong> <%= "#{@submission.user.username}" %> </strong>
 
 <h4>Description</h4>
-<p><%= @submission.description %></p>
+<p><%= @submission.comments[0].body %></p>
```
Comment (kyjoya) [inline app/views/submissions/show.html.erb]: I would save this to an instance variable @description in the controller and do this logic there.

File: app/views/submissions/show.html.erb (hunk 2 above)
Comment (kiltedDev) [reply app/views/submissions/show.html.erb:93]: The body of the first rating is being used as the description of the submission.  dropping it removes the double.

**Pull request merged**